src/gwt/src/org/rstudio/core/client/command/AppCommand.java

```diff
@@ -219,7 +219,12 @@ public boolean hasCommandHandlers()
 
    public void setEnabled(boolean enabled)
    {
-      if (enabled != enabled_)
+      setEnabled(enabled, false);
+   }
+   
+   public void setEnabled(boolean enabled, boolean force)
+   {
+      if (force || enabled != enabled_)
```

Comment:
For context: RStudio maintains command "state" both on the GWT side, and also on the Desktop side for RStudio Desktop. (This indicates whether the Save command should be enabled in e.g. the File menu.)

I was seeing some cases where the client-side save command was marked as enabled, but the desktop-side save command was still marked as disabled. Having `force` here allows us to ensure that the client + desktop command states are sync'ed in the case where this happens.